web/e2e/page-objects/flows/manageFlows.ts

```diff
@@ -216,14 +216,14 @@ export class ManageFlows extends AppPage {
    */
 
   async clickPaginationDropDown() {
-    return await element(by.id("mat-select-0")).click();
+    return await element(by.id("mat-select-arrow")).click();
   }
 
-  itemsPerPage(value: number) {
-    return element(by.css(`mat-option[ng-reflect-value="${value}"]`));
+  itemsPerPage(value: string) {
+    return element(by.cssContainingText("mat-option .mat-option-text", value));
```

Comment:
isn't the same thing? it goes to the same selector, it doesn't need to be changed

Reply:
Is ng-reflect-value attribute of web element? I didn't find it. Was it there before?

Reply:
Yes it was there when I first added it back in March. Lot of changes since then. Its not there anymore. Changing the type to string has impact in other places where itemsPerPage is used. For instance selectItemsPerPage and the corresponding tests. If string works then pls change all the affected places.

Reply:
Hmm...you can't see it? I'm using latest develop:
<img width="1098" alt="Screen Shot 2019-04-26 at 11 15 00 AM" src="https://user-images.githubusercontent.com/7096668/56827802-ab0e8180-6814-11e9-87f4-3f73deebfd26.png">